{"name": "Logout button is clickable after successful login", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000100d5e668 cxxbridge1$str$ptr + 2723108\n1   chromedriver                        0x0000000100d568dc cxxbridge1$str$ptr + 2690968\n2   chromedriver                        0x00000001008aa714 cxxbridge1$string$len + 90428\n3   chromedriver                        0x00000001008f17c0 cxxbridge1$string$len + 381416\n4   chromedriver                        0x0000000100932de8 cxxbridge1$string$len + 649232\n5   chromedriver                        0x00000001008e59c8 cxxbridge1$string$len + 332784\n6   chromedriver                        0x0000000100d2228c cxxbridge1$str$ptr + 2476360\n7   chromedriver                        0x0000000100d25520 cxxbridge1$str$ptr + 2489308\n8   chromedriver                        0x0000000100d03a78 cxxbridge1$str$ptr + 2351412\n9   chromedriver                        0x0000000100d25da8 cxxbridge1$str$ptr + 2491492\n10  chromedriver                        0x0000000100cf4d6c cxxbridge1$str$ptr + 2290728\n11  chromedriver                        0x0000000100d45d74 cxxbridge1$str$ptr + 2622512\n12  chromedriver                        0x0000000100d45f00 cxxbridge1$str$ptr + 2622908\n13  chromedriver                        0x0000000100d56528 cxxbridge1$str$ptr + 2690020\n14  libsystem_pthread.dylib             0x00000001877732e4 _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000018776e0fc thread_start + 8\n\n", "trace": "  File \"/Users/aniha/PycharmProjects/ZenPortalLogin-PythonBehave/.venv/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/aniha/PycharmProjects/ZenPortalLogin-PythonBehave/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps.py\", line 56, in step_impl\n    context.page.close_popup()\n  File \"/Users/aniha/PycharmProjects/ZenPortalLogin-PythonBehave/pages/login_page.py\", line 23, in close_popup\n    self.wait.until(EC.element_to_be_clickable(Locators.close_popup_button)).click()\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/aniha/PycharmProjects/ZenPortalLogin-PythonBehave/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given the user is logged in with valid credentials", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n0   chromedriver                        0x0000000100d5e668 cxxbridge1$str$ptr + 2723108\n1   chromedriver                        0x0000000100d568dc cxxbridge1$str$ptr + 2690968\n2   chromedriver                        0x00000001008aa714 cxxbridge1$string$len + 90428\n3   chromedriver                        0x00000001008f17c0 cxxbridge1$string$len + 381416\n4   chromedriver                        0x0000000100932de8 cxxbridge1$string$len + 649232\n5   chromedriver                        0x00000001008e59c8 cxxbridge1$string$len + 332784\n6   chromedriver                        0x0000000100d2228c cxxbridge1$str$ptr + 2476360\n7   chromedriver                        0x0000000100d25520 cxxbridge1$str$ptr + 2489308\n8   chromedriver                        0x0000000100d03a78 cxxbridge1$str$ptr + 2351412\n9   chromedriver                        0x0000000100d25da8 cxxbridge1$str$ptr + 2491492\n10  chromedriver                        0x0000000100cf4d6c cxxbridge1$str$ptr + 2290728\n11  chromedriver                        0x0000000100d45d74 cxxbridge1$str$ptr + 2622512\n12  chromedriver                        0x0000000100d45f00 cxxbridge1$str$ptr + 2622908\n13  chromedriver                        0x0000000100d56528 cxxbridge1$str$ptr + 2690020\n14  libsystem_pthread.dylib             0x00000001877732e4 _pthread_start + 136\n15  libsystem_pthread.dylib             0x000000018776e0fc thread_start + 8\n\n", "trace": "  File \"/Users/aniha/PycharmProjects/ZenPortalLogin-PythonBehave/.venv/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/aniha/PycharmProjects/ZenPortalLogin-PythonBehave/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps.py\", line 56, in step_impl\n    context.page.close_popup()\n  File \"/Users/aniha/PycharmProjects/ZenPortalLogin-PythonBehave/pages/login_page.py\", line 23, in close_popup\n    self.wait.until(EC.element_to_be_clickable(Locators.close_popup_button)).click()\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/aniha/PycharmProjects/ZenPortalLogin-PythonBehave/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1749643939019, "stop": 1749643953806}, {"name": "Then the logout button should enabled", "status": "skipped", "start": 1749643953812, "stop": 1749643953812}], "start": 1749643939018, "stop": 1749643953812, "uuid": "bb0a6511-b838-42f7-911e-ee9565bd39ac", "historyId": "1292f80c29903a8c6d0a27ddf8b74547", "fullName": "Login functionality for Guvi-Zen portal: Logout button is clickable after successful login", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login functionality for Guvi-Zen portal"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}